_posts/2024-07-02-schematic-review-checklist.md

```diff
@@ -106,7 +106,7 @@ Here's my current checklist. I didn't list all the stories as above (this would'
 
 - [ ] How will you (a firmware engineer) attach a debugger to the board? (By soldering is not usually a good answer).[5](#story-5-too-much-or-to-little-debug)
 - [ ] Are non-essential, but helpful, GPIOs exposed for a debugger, such as SWO or ETM?
-- [ ] Do test points exist for power rails, analog signals, and busses?
+- [ ] Do test points exist for power rails, analog signals, and buses?
 
 
 ## Conclusion
```

Comment:
I don't want to downplay what you've provided -- we know from the intro this isn't an exhaustive list. Instead of `This is the start of my checklist for schematic reviews.`, how about `This checklist captures the most important items I have learned to ask about as a firmware developer in schematic reviews."`? @schultetwin